cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py

```diff
@@ -14,17 +15,29 @@
     from cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml import (
         GenerateAndLoadDataFromYaml,
     )
+    from snowfakery import data_generator_runtime
```

Comment:
These no longer need to be imported conditionally.

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1659/commits/c9f7e87ed50d4bcf2e2797c54b8dd01ab1e41dbc